src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -2473,22 +2466,9 @@ private TriggerableSelector<Rectangle> backgroundImageBorderSelector
         [EditorBrowsable(EditorBrowsableState.Never)]
         public virtual void ApplyStyle(ViewStyle viewStyle)
         {
-            if (null == viewStyle)
-            {
-                return;
-            }
-
-            if (this.viewStyle == viewStyle)
-            {
-                return;
-            }
+            if (null == viewStyle || this.viewStyle == viewStyle) return;
 
-            if (null != this.viewStyle)
-            {
-                simpleBinding.Clear();
-            }
-
-            this.viewStyle = viewStyle;
+            this.ViewStyle.CopyFrom(viewStyle);
```

Comment:
There is a case that a component getting extended style (LottieButtonStyle for a Button, for example),
so the code `this.ViewStyle.CopyFrom(viewStyle)` can lost extended data.
Please fix it like below code,
```
this.viewStyle =  viewStyle.Clone();
```

Reply:
Fixed